### PR TITLE
Document how to run Elixir services & add Elixir services to Tilt

### PR DESCRIPTION
**Pull request by pkanal**
## Which problem is this PR solving?
- Fixes #123 

There is an example Elixir app but running it was not documented and it wasn't available to run through Tilt. This PR adds docs for how to run the Elixir services and makes Elixir services available to run through Tilt.

## Short description of the changes
- Added `elixir` services to Tilt
- Added README instructions for all the different ways to run Elixir services
- Edited `editorconfig` because the line endings were messing with `.tool-versions` 😭 

## Expected Outcome(s)
- [x] Should be able to run Elixir services with Tilt by running `tilt up elixir`
- [x] Should be able to run Elixir services with Docker by running `docker-compose up`
- [x] Should be able to run individual Elixir services by following instructions
- [x] Elixir services should send data to Honeycomb

### CODE REVIEW

**Comment by cartermp** [comment]
Mmmm, an elixir of EGS. I may crib some of this stuff to make some upstream docs improvements for erlang/elixir, too

File: Tiltfile [hunk 1]
```diff
@@ -231,6 +231,40 @@ def launch_node_name_service(auto_init=True):
...
   launch_node_svc("year-node", dirname="node/year-service", auto_init=auto_init)
 
+def launch_elixir_svc(name, dirname="", cmd="", auto_init=True):
+    '''
+    Starts a single Node service.
+
+    Parameters:
+    name: used to display the name of the process in the tilt tab
+    dirname: (optional) directory name in which to run `npm start` defaults to 'name'
+    flags: (optional) any additional flags to add to the command line
```
Comment (MikeGoldsmith) [inline Tiltfile:241]:
```suggestion
    Starts a single Elixir service.

    Parameters:
    name: used to display the name of the process in the tilt tab
    dirname: (optional) directory name in which to run the app defaults to 'name'
    flags: (optional) any additional flags to add to the command line

```

**Review by MikeGoldsmith — APPROVED** [review]
Just a small copy/paste typo - otherwise looks good 👍🏻

**Comment by pkanal** [comment]
> Mmmm, an elixir of EGS. I may crib some of this stuff to make some upstream docs improvements for erlang/elixir, too

Let me know if I can help with this at all!

**Pull request merged**